config/rbac/kustomization.yaml

```diff
@@ -0,0 +1,10 @@
+resources:
+- service_account.yaml
+- role.yaml
+- role_binding.yaml
+- leader_election_role.yaml
+- leader_election_role_binding.yaml
```

Comment:
I thought you said [we don't enable leader election](https://github.com/IBM/ibm-block-csi-operator/pull/195#discussion_r689092818)

Reply:
I don't see any change

Reply:
removed them now, sorry

Reply:
so why do I still see [new additions](https://github.com/IBM/ibm-block-csi-operator/blob/cbf6aa2cb1bfb4a860cf607c6d891f3ea8af8225/main.go#L67-L71) e.g. leader election?
do you want to enable them and get it over with?

Reply:
CSI-3314

Reply:
what about the other said additions?
where are they coming from? should we use them?

Reply:
removed them
done
CSI-3344
CSI-3343